lib/ASRouterTargeting.jsm

```diff
@@ -176,6 +179,12 @@ const TargetingGetters = {
   },
   get totalBookmarksCount() {
     return TotalBookmarksCountCache.getTotalBookmarksCount;
+  },
+  get firefoxVersion() {
+    return parseInt(AppConstants.MOZ_APP_VERSION, 10);
+  },
+  get geoCountry() {
+    return Services.prefs.getStringPref(SEARCH_REGION_PREF, "");
```

Comment:
I wonder if we should do `Services.prefs.getStringPref(SEARCH_REGION_PREF, "") || "ZZ"` when nothing is found like the server side filter does?